StickyHeaders/src/main/java/io/doist/recyclerviewext/sticky_headers/StickyHeadersLinearLayoutManager.java

```diff
@@ -296,6 +370,22 @@ protected void bindStickyHeader(@NonNull RecyclerView.Recycler recycler, int pos
         recycler.bindViewToPosition(mStickyHeader, position);
         mStickyHeaderPosition = position;
         measureAndLayoutStickyHeader();
+
+        // If we have a pending scroll wait until the end of layout and scroll again.
+        if (mPendingScrollPosition != RecyclerView.NO_POSITION) {
+            final ViewTreeObserver vto = mStickyHeader.getViewTreeObserver();
+            vto.addOnGlobalLayoutListener(new ViewTreeObserver.OnGlobalLayoutListener() {
+                @Override
+                public void onGlobalLayout() {
+                    vto.removeOnGlobalLayoutListener(this);
+
+                    if (mPendingScrollPosition != RecyclerView.NO_POSITION) {
+                        scrollToPositionWithOffset(mPendingScrollPosition, mPendingScrollOffset);
+                        setPendingScroll(RecyclerView.NO_POSITION, 0);
+                    }
+                }
+            });
+        }
```

Comment:
I'm a bit confused by this. Since we measure _and_ layout the sticky header manually, before this call, does it make sense to register in this listener? Can't we just run `scrollToPositionWithOffset` directly?